synapse/cores/common.py

```diff
@@ -284,13 +387,11 @@ def _initCortexConfSetPre(self):
         self.onConfOptSet('caching', self._onSetCaching)
 
     def _initCortexConfSetPost(self):
-        self._initCoreFifos()
 
-        # Setup the membranes conf handler and initialize membranes we may have
-        self.onConfOptSet('membranes', self._onSetMembranes)
-        valu = self.getConfOpt('membranes')
-        if valu:
-            self.fire('syn:conf:set:%s' % 'membranes', valu=valu)
+        # Configure our auth data:
+        if self.getConfOpt('auth:en'):
+            self._onSetAuthEn({'auth:en': 1})
```

Comment:
What if you manually set the config valu to 0? Will this still set it to 1?

Reply:
fixed this - should have passed in `True`